plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -281,13 +378,33 @@ export const resolvers = (): IResolvers => ({
     ): Observable<JobConnection | null> {
       return Observable.of(null);
     },
-    job(_obj = {}, _args: GeneralArgs, _context = {}): Observable<Job | null> {
-      return Observable.of(null);
+    job(_obj = {}, args: GeneralArgs, _context = {}): Observable<Job | null> {
+      // TODO: We should not need to do the casting here
+      const { id } = args as JobQueryArgs;
```

Comment:
I put this comment a little bit down here but its for the `_obj`.

I believe the name we should use here is `_parent`. I know appolo names it obj. but the definition they gave is even: 

```
The object that contains the result returned from the resolver on the parent field, or, in the case of a top-level Query field, the rootValue passed from the server configuration. This argument enables the nested nature of GraphQL queries.
```